### PR TITLE
Playing with some formatting

### LINKED ISSUE #452: Remove module details below the schedule
@avahoffman I've been wondering if we need the module details below the schedule since the links are the same as the table... maybe we just need to word details in the table? or is there something else you want to convey about that section

_Originally posted by @carriewright11 in https://github.com/jhudsl/intro_to_r/issues/444#issuecomment-1662617861_

### PR DESCRIPTION
**Pull request by avahoffman**
Syllabus
- Got rid of the lines, it looked a little cluttered (I think I added them orgiinally lol)
- Added a table for the grading breakdown and homework

Schedule
- Removed detailed schedule (kind of redundant, resolves #452 )

Resources
- Split 'Tidyverse Resources' into getting started and Help (learners might not feel comfortable with the term tidyverse yet)
- Combined courses + conferences
- Try expanding approach for resources in case learners want to see more than one at once

Help
- changed to FAQ/Help so they know they can find answers to specific questions 🙂 
- Removed code formatting so TOC doesn't look weird
- added refs to backticks and `pull()` resources

### CODE REVIEW

**Comment by carriewright11** [comment]
might want to capitalize: 
[video](https://www.youtube.com/watch?v=we6vwB7DsNU) for PC users who want to see how to move files around (especially from downloads)
[video](https://www.youtube.com/watch?v=Ao9e0cDzMrE) for Mac users who want to see how to move files around (especially from downloads)

File: resources.Rmd [hunk 1]
```diff
@@ -8,26 +9,23 @@ library(dplyr)
...
-<br>
-
 
-### **Help getting started**
+<details><summary> <span style = "color: #5383bb;"> ► **Help Getting Started**</span></summary><br>
 
 - [R reference card](http://cran.r-project.org/doc/contrib/Short-refcard.pdf) 
 - [R jargon](https://link.springer.com/content/pdf/bbm%3A978-1-4419-1318-0%2F1.pdf) 
 - [R terminology](https://cran.r-project.org/doc/manuals/r-release/R-lang.pdf)
+- [What is the Tidyverse?](https://www.tidyverse.org/packages/)
```
Comment (carriewright11) [inline resources.Rmd:21]:
```suggestion
- [What is the Tidyverse/what packages are in it?](https://www.tidyverse.org/packages/)   
```

Comment (carriewright11) [reply resources.Rmd:21]: I know we have this on the help page- but might make sense here?

**Comment by carriewright11** [comment]
I think this is great and we are really getting somewhere. The help page looks great. Still have those "i" icons - which I am happy to help figure out how to remove. - I also wonder if we should have categories within those sections. (need to think about that though)

For the resource page: I also wonder if the help getting started and extra help sections should maybe be expanded for the resource page? just cause those are so fundamental and broad and people might not know that they want to look at those?

**Comment by avahoffman** [comment]
@carriewright11 
>  Still have those "i" icons - which I am happy to help figure out how to remove.
This is actually part of the output:
image
> I also wonder if we should have two categories to break it up - like common issues and common errors?
Sure, will do!
> For the resource page: I wonder if the help getting started and extra help sections should maybe be expanded for the resource page? just cause those are so fundamental and broad and people might not know that they want to look at those? (we would need to change the directions a tad)
Love this idea!

**Comment by carriewright11** [comment]
sorry I overwrote your comment! 
I am glad you like the ideas

**Pull request merged**